roles/confluent.common/molecule/default/verify.yml

```diff
@@ -0,0 +1,44 @@
+---
+- name: Verify
+  hosts: all
+  tasks:
+    - name: Get Package Facts
+      package_facts:
+        manager: auto
+
+    - name: Assert java installed - Redhat
+      assert:
+        that: "ansible_facts.packages['java-1.8.0-openjdk'] is defined"
+      when: ansible_os_family == 'RedHat'
+
+    - name: Assert java installed - Debian
+      assert:
+        that: "ansible_facts.packages['openjdk-8-jdk'] is defined"
+      when: ansible_os_family == "Debian"
+
+    - name: Check if jolokia jar exists
+      stat:
+        path: /opt/jolokia/jolokia.jar
+      register: jolokia_jar
+
+    - name: Assert jolokia jar downloaded
+      assert:
+        that: "jolokia_jar.stat.exists == True"
+
+    - name: Check if prometheus jar exists
+      stat:
+        path: /opt/prometheus/jmx_prometheus_javaagent.jar
+      register: prometheus_jar
+
+    - name: Assert prometheus jar not downloaded
+      assert:
+        that: "prometheus_jar.stat.exists == False"
+
+    - name: Check if custom repofile exists
+      stat:
+        path: /etc/yum.repos.d/custom-confluent.repo
+      register: custom_repofile
+
+    - name: Assert custom repofile created
+      assert:
+        that: "custom_repofile.stat.exists == False"
```

Comment:
This looks like it rechecks the custom-vars steps. Is there a need to repeat it?

Reply:
i pretty much chose to put the opposite vars then the defaults in custom-vars scenario. so youll see these checks are the opposite